doc8/action.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: "doc8 for Py3"
+author: "Ben Alkov <ben.alkov@redhat.com>"
+description: "GitHub action for doc8 using Python 3."
+inputs:
+  path:
+    description: >-
+      Path containing reStructuredText files to check. Can be a single file, wildcard, etc.
+    required: true
+    default: 'docs/'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args:
+    - ${{ inputs.path }}
```

Comment:
Do the args override or extend the default `CMD`?

Reply:
Override

Reply:
It's why you keep ENTRYPOINT and CMD separate like that

Reply:
Right, but that means the `--max-line-length=100` argument is lost, no?

Reply:
ARGH, I wasn't thinking that through. You're right, `docker run` args override CMD.

Reply:
Other Dockerfile fixes, along with README updates, are addressed in #13